### ISSUE TITLE
Biome encountered an unexpected error [nvim | mason-lspconfig]

### ISSUE THREAD

**Issue by ancientjpeg** [biomejs/biome#2718]
### Environment information

```block
CLI:
  Version:                      1.7.2
  Color support:                true

Platform:
  CPU Architecture:             aarch64
  OS:                           macos

Environment:
  BIOME_LOG_DIR:                unset
  NO_COLOR:                     unset
  TERM:                         "screen-256color"
  JS_RUNTIME_VERSION:           "v20.12.2"
  JS_RUNTIME_NAME:              "node"
  NODE_PACKAGE_MANAGER:         unset

Biome Configuration:
  Status:                       Loaded successfully
  Formatter disabled:           false
  Linter disabled:              false
  Organize imports disabled:    false
  VCS disabled:                 true

Workspace:
  Open Documents:               0
```

### What happened?

1. Attempted to startup biome as an lsp in neovim
2. Noticed this error in the logs

I'm using biome through `mason-lspconfig`. I'm currently having some trouble with my setup, but not sure if it's related to this bug. I'm just reporting this as I caught it in the logs.

### Expected result

No error should be thrown.

### Code of Conduct

- [X] I agree to follow Biome's Code of Conduct

**Comment by ematipico** [comment]
Probably related to https://github.com/biomejs/biome/issues/2683

**Comment by Conaclos** [comment]
Hi @ancientjpeg.
I think the bug was fixed in the patch version released today (v1.7.3).
Could you confirm that you no longer see the bug you reported?
Thanks!
